include/language.php

```diff
@@ -392,57 +305,89 @@ public function __construct( array $language_data ) {
 		foreach ( $language_data as $prop => $value ) {
 			$this->$prop = $value;
 		}
+
+		$this->term_id = $this->term_props['language']['term_id'];
 	}
 
 	/**
-	 * Returns a language term property value (term ID, term taxonomy ID, or count).
+	 * Throws a depreciation notice if someone tries to get one of the following properties:
+	 * `term_taxonomy_id`, `count`, `tl_term_id`, `tl_term_taxonomy_id` or `tl_count`.
 	 *
 	 * @since 3.4
 	 *
-	 * @param string $taxonomy_name Name of the taxonomy.
-	 * @param string $prop_name     Name of the property: 'term_taxonomy_id', 'term_id', 'count'.
-	 * @return int
+	 * @param string $property Property to get.
+	 * @return mixed Required property value.
```

Comment:
Can't we be more precise?

Reply:
Good idea, done in b0ac5cd. I hope I didn't miss anything.

Reply:
Good thinking 👍